notebooks/yolov8-optimization/yolov8-obb.ipynb

```diff
@@ -49,8 +49,8 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        if device.value != "CPU":
</pre><p>this is no need, quantization inside infer model on cpu, only before compilation reshaping required</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1934/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>removed</p>